aicsimageio/readers/scn_reader.py

```diff
@@ -0,0 +1,245 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple, Union
+
+import dask.array as da
+import numpy as np
+import xarray as xr
+import zarr
+from fsspec.implementations.local import LocalFileSystem
+from tifffile.tifffile import TiffFile, TiffFileError, ZarrTiffStore, xml2dict
+
+from .. import constants, exceptions, types
+from ..dimensions import DimensionNames
+from ..utils import io_utils
+from .reader import Reader
+
+if TYPE_CHECKING:
+    from fsspec.spec import AbstractFileSystem
+
+
+class SCNReader(Reader):
+    """Read Leica SCN files using tifffile.
+
+    Parameters
+    ----------
+    image : Path or str
+        path to file
+
+    Raises
+    ------
+    exceptions.UnsupportedFileFormatError
+        If the file is not supported by ND2.
+    """
+
+    @staticmethod
+    def _is_supported_image(fs: AbstractFileSystem, path: str, **kwargs: Any) -> bool:
+        try:
+            with fs.open(path) as open_resource:
+                with TiffFile(open_resource) as rdr:
+                    return bool(rdr.scn_metadata)
+
+        except (TiffFileError, TypeError):
+            return False
+
+    def __init__(self, image: types.PathLike, scene: Optional[Union[int, str]] = None):
```

Comment:
Not sure I agree with adding the `scene` parameter.... isn't that just shorting the `img = ... img.set_scene()`?